pkg/components/validator.go

```diff
@@ -114,6 +120,10 @@ func (c *ValidatorConfig) EnabledPluginsHaveRules() (bool, []string) {
 		invalidPlugins = append(invalidPlugins, "Azure")
 		// invalidPlugins = append(invalidPlugins, c.AzurePlugin.Validator.PluginCode())
 	}
+	if c.MaasPlugin.Enabled && c.MaasPlugin.Validator.ResultCount() == 0 {
+		// invalidPlugins = append(invalidPlugins, c.MaasPlugin.Validator.PluginCode())
```

Comment:
PluginCode should be usable for MAAS: https://github.com/validator-labs/validator-plugin-maas/blob/main/api/v1alpha1/maasvalidator_types.go#L151

Reply:
Please see if it's ok to update the Azure one too 🙏🏼

Reply:
updating OCI and vSphere as well